openssl-sys/build/find_normal.rs

```diff
@@ -25,16 +25,11 @@ fn resolve_with_wellknown_homebrew_location(dir: &str) -> Option<PathBuf> {
     // Check up default aarch 64 Homebrew installation location first
     // for quick resolution if possible.
     //  `pkg-config` on brew doesn't necessarily contain settings for openssl apparently.
-    let mut version_dir = dir.to_owned();
-    version_dir.push_str("@1.1");
-    let homebrew = Path::new(&version_dir);
+    let homebrew = Path::new(dir).join("opt/openssl@1.1");
```

Comment:
After these changes it will no longer find an OpenSSL 1.0.2 install (under `/usr/local/opt/openssl`).

Reply:
This is intentional as only OpenSSL 1.1 is provided by Homebrew [as of November 2019](https://brew.sh/2019/11/27/homebrew-2.2.0/), however can split into a separate PR if this is worth discussing to focus on the initial changes in this PR :)

Reply:
Oh nice, I was not aware of that!

Reply:
Do you know if that change actively removed existing 1.0.2 installs or just prevented new ones? If it's the former, there is a chance someone who was "grandfathered" in could have their builds broken by this change.

Reply:
Hmm, although I do not primarily use Homebrew myself, I believe that running `brew upgrade` would cause it to upgrade to the latest version, and it seems unlikely that someone would have skipped updating their packages for that long a period of time.

Reply:
openssl@1.1 was actually a separate package from openssl, so I'm not sure the automatic upgrade would have happened. In any case though, anyone in that situation really needs to bump to a version that's actually supported, so we can probably do this anyway!